pkg/apis/config/v1alpha1/resourceinterpretercustomization_types.go

```diff
@@ -36,6 +36,8 @@ const (
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:resource:path=resourceinterpretercustomizations,scope="Cluster",shortName=ric,categories={karmada-io}
 // +kubebuilder:storageversion
+// +kubebuilder:printcolumn:JSONPath=`.spec.target.apiVersion`,name="Target-API-Version",type=string
+// +kubebuilder:printcolumn:JSONPath=`.spec.target.kind`,name="Target-Kind",type=string
```

Comment:
```suggestion
// +kubebuilder:printcolumn:JSONPath=`.spec.target.apiVersion`,name="TARGET-API-VERSION",type=string
// +kubebuilder:printcolumn:JSONPath=`.spec.target.kind`,name="TARGET-KIND",type=string
```
I guess that's what @XiShanYongYe-Chang and @chaunceyjiang mean.
LGTM. 
But the previous APIs don't follow this rule ;) :) :)

Reply:
So, I don't have strong opinions on this.
@XiShanYongYe-Chang Are there any documents about best practices or something?

Reply:
I don't have this best practice.